README.adoc

```diff
@@ -61,7 +61,9 @@ https://asciinema.org/a/wVkVgUrO7PGR5CYBFbHB5fFDn[image:https://asciinema.org/a/
 [[contributing]]
 == Community, discussion, contribution, and support
 
-*Discussions:* For any discussions or help with `odo`, you can use our link:https://groups.google.com/forum/#!forum/odo-users[Google group] or open a link:https://github.com/openshift/odo/issues[GitHub issue].
+*Chat:* Join us on link:https://join.slack.com/t/openshiftdo/shared_invite/enQtODUwMjIzMzE2MjYyLWM3YjlkNTViOGNjZGQ5YjFlMzc2N2RhZjVmMDQ5NWY5ZDQ3NzMzZjYxMTE1MjY5OTVmZjgxZWMyMGRmOTc4Y2I[Slack].
+
+*Discussions:* For any developmental discussions with `odo`, use our link:https://groups.google.com/forum/#!forum/odo-users[Google group].
```

Comment:
For discussions related to odo development, it should be odo-dev, for getting help and ask questions about using odo it should be odo-users